app/views/create.html

```diff
@@ -0,0 +1,50 @@
+<div class="row">
+  <div class="col-xs-12">
+    <h1>Postcard Create</h1>
+    <hr>
+  </div>
+</div>
+<div class="row postcard-create">
+  <div class="col-xs-12 col-sm-5">
+    <form>
+      <div class="form-group">
+        <label class="dashboard-control-label" for="description">Description</label>
+        <input class="form-control" id="description" name="description" placeholder="this is a postcard">
+      </div>
+
+      <div class="form-group">
+        <label class="dashboard-control-label" for="front">Front URL<span class="required-asterisk">*</span></label>
+        <input class="form-control" id="front" name="front" placeholder="https://lob.com/postcardfront.pdf">
```

Comment:
nit: don't know what the reqs for miyagi are, but I'd put the `lob.com` part of the URL as a config variable like `LOB_URL` so you can more easily re-use it, and it will differ in staging/prod.